resources/js/Contexts/DraftGalleriesContext.tsx

```diff
@@ -46,6 +48,14 @@ export const DraftGalleriesContextProvider = ({ children }: ProviderProperties):
         await loadWalletDrafts();
     };
 
+    useEffect(() => {
+        const isMyGalleryDraftPage = route().current("my-galleries", { draft: true });
+
+        if (isTruthy(drafts) && drafts.length === 0 && isMyGalleryDraftPage) {
```

Comment:
```suggestion
        if (isMyGalleryDraftPage && isTruthy(drafts) && drafts.length === 0) {
```

tiny performance improvment